get_dem.py

```diff
@@ -126,7 +126,7 @@ def do_computations(dem: np.ndarray) -> None:
     square_dem = dem[:min_edge, :min_edge]
 
     # Multi-core section
-    np.dot(np.linalg.pinv(square_dem), square_dem)
+    np.dot(np.linalg.inv(square_dem), square_dem)
```

Comment:
Because we've already started benchmarking with the `np.pinv()` function, I don't want to derail that process and have to redo the benchmarking.

That said, having this be `np.inv()` is a quick litmus test for whether the bug is resolved or not, so I can see why it's useful to have in this PR.

These seem to be two possible paths forward:
1) Leave as `np.inv()` in this PR, but do not merge this PR until after this round of benchmarking is complete. (maybe 1-3 weeks delay?)
2) Revert to `np.pinv()` in this PR so that we can merge it asap. On the plus side, this approach means that we do not need to redo the existing benchmarking results. However, the cost is that we'll need to manually hack the code to get our litmus test of whether the "fix" worked or not... which means it's very hard to test in DPS.

@arthurduf @chuckwondo  Thoughts? Preferences?